modules/access-html-elements.js

```diff
@@ -0,0 +1,30 @@
+// js to access html elements
+// eslint-disable-next-line max-classes-per-file
```

Comment:
[ OPTIONAL ] I highly recommend not disabling the linters because you should just solve the issues not disable it so please try to solve it 😊💪🏻💯